helm/lib/cortex-postgres/templates/configmap.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2025 SAP SE
+# SPDX-License-Identifier: Apache-2.0
+
+# These will always be mounted into the postgres container since the mount
+# is done unconditionally through the helm values of the bitnami postgresql chart.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.postgresql.fullnameOverride }}-pre-upgrade-script
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+data:
+  pre-upgrade.sh: |
+  {{- range .Files.Lines "files/pre-upgrade.sh" }}
+    {{ . }}
+  {{- end }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.postgresql.fullnameOverride }}-post-upgrade-script
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
```

Comment:
These labels are managed by Helm and don't need to be added as they're overwritten anyway.